targets/TARGET_STM/TARGET_STM32L0/serial_device.c

```diff
@@ -49,25 +49,58 @@ static uart_irq_handler irq_handler;
  * INTERRUPTS HANDLING
  ******************************************************************************/
 
-static void uart_irq(int id)
+static int8_t uart_pointer_get_index(int uart_base)
 {
-    UART_HandleTypeDef * huart = &uart_handlers[id];
-    
-    if (serial_irq_ids[id] != 0) {
-        if (__HAL_UART_GET_FLAG(huart, UART_FLAG_TXE) != RESET) {
-            if (__HAL_UART_GET_IT(huart, UART_IT_TXE) != RESET) {
-                irq_handler(serial_irq_ids[id], TxIrq);
+    uint8_t index = 0;
+#if defined(USART1_BASE)
+    if (uart_base == USART1_BASE) return index;
```

Comment:
can you align the new changes please (see https://docs.mbed.com/docs/mbed-os-handbook/en/latest/cont/code_style/)